wscript

```diff
@@ -24,6 +24,7 @@ def configure(conf):
 
 def build(bld):
     bld.recurse(DIRS)
+    enableWafUnitTests(bld)
```

Comment:
Does this always run all unittests after building or is there a flag I set at configure time? How does this compare with the existing "--unittests" flag (I honestly don't know how the existing flag works, I just know that Jenkins sets it)?

Reply:
The previous behavior for the `--unittests` flag was to mark unit tests as tests when passed. As far as I can tell, this causes waf to internal run the tests, but not tell you if they passed or not. I've just removed it.

The new behavior, if you build the first time, it will automatically run all of the tests and return non-zero on failure. Then if you build again, it will only re-run the tests that needed to recompile. (Unless you pass `--alltests`). As far as I know, projects that use coda-oss jump over this top-level wscript, so their behavior will be unaffected.